Document vagrant beaker backend (#33)
voxpupuli/voxpupuli-acceptance: README.md

Other common environment variables:

> * `BEAKER_HYPERVISOR` defaults to `docker`, can be set to `vagrant_libvirt` or `vagrant` (which will use Virtualbox under the hood)

Reviewer: Shorter version?
```suggestion
* `BEAKER_HYPERVISOR` defaults to `docker`, can be set to `vagrant_libvirt` or `vagrant` (using [VirtualBox](https://www.virtualbox.org/))
```
